docs/spec/data-plane.md

```diff
@@ -91,6 +91,25 @@ delivered.
 CloudEvents received by Sink MAY have
 [Distributed Tracing Extension Attribute](https://github.com/cloudevents/spec/blob/v1.0/extensions/distributed-tracing.md).
 
+### HTTP Attributes
+
+An event sender, including Source and Broker and Channel, MUST include
+all the non-default HTTP attributes with header key `K-Eventing-Http-Attr` in
+every event delivery to specify
+what HTTP features it can support. Without a certain attribute, the receiver
+MUST assume the corresponding HTTP feature is not supported. A list of HTTP
+attributes can be found below:
+
+| Attributes         | HTTP Feature Description                                                  |
+| ------------------ | ------------------------------------------------------------------------- |
+| `callable`         | If the event sender supports event reply in HTTP response.                |
+
+An example is that a broker supporting event reply MUST send events with
+an additional header `K-Eventing-Http-Attr: ["callable"]` so that the sink connected
+to the broker knows event replies will be accepted. While a source
```

Comment:
s/`broker`/`Broker`/ 

But this is wrong, sinks and sources have no awareness of Broker, they only understand the data plane contract of their object refs or what they implement at the moment. If this was added, they would also understand what the caller is assuming the runtime contract is for the caller and this request.